test/wave1.test.js

```diff
@@ -1,52 +1,56 @@
-const axios = require('axios');                    // Import axios.
+const axios = require('axios'); // Import axios.
 const MockAdapter = require('axios-mock-adapter'); // This is kind of like VCR.
-const result = require('../src/result.js');        // Import result handling.
+const result = require('../src/result.js');
+ // Import result handling.
 const setHandlers = result.setHandlers;
 
-//b Import our function(s) for testing.
+// b Import our function(s) for testing.
```

Comment:
Oops, that `b` was just a typo.

```suggestion
// Import our function(s) for testing.
```